arrows/core/detected_object_set_output_kw18.cxx

```diff
@@ -33,12 +33,17 @@
 #include <vital/vital_foreach.h>
 #include <vital/util/tokenize.h>
 
-#include <atomic>
 #include <memory>
 #include <vector>
 #include <fstream>
 #include <time.h>
 
+#if (__GNUC__ == 4 && __GNUC_MINOR__ < 5 )
+  #include <cstdatomic>
```

Comment:
I believe this was added to support gcc 4.4 (from older CentOS). I don't believe it is needed for current systems.